### PR TITLE
[risk=no] Fix 'Whole Genome Sequence' text in cohort actions test

### PR DESCRIPTION
**Pull request by dolbeew**
- Fixes test with updated name for cohort search item
- Bump cdr to `synth_r_2019q4_17` for Synthetic Dataset v3 in test env

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: e2e/tests/cohorts/cohort-group-actions.spec.ts [hunk 1]
```diff
@@ -186,9 +186,9 @@ describe('Build cohort page actions', () => {
     expect((await group1.findGroupCriteriaList()).length).toBe(2);
     totalCount = await cohortBuildPage.getTotalCount(); // New Total Count by 2 criteria.
 
-    // Rename Criteria: rename "Whole Genome Variant" to "NDA Sets".
+    // Rename Criteria: rename "Whole Genome Sequence" to "NDA Sets".
     const newCriteriaName = 'DNA Sets';
-    await group1.editCriteriaName('Whole Genome Variant', newCriteriaName);
+    await group1.editCriteriaName('Whole Genome Sequence', newCriteriaName);
```
Comment (NehaBroad) [inline e2e/tests/cohorts/cohort-group-actions.spec.ts:191]: This will still need a change in DB right?

**Comment by NehaBroad** [comment]
Can you update text-label.ts as well in this PR

**Comment by NehaBroad** [comment]
nvm i think thats already updated as part of the other PR

File: e2e/tests/cohorts/cohort-group-actions.spec.ts (hunk 1 above)
Comment (dolbeew) [reply e2e/tests/cohorts/cohort-group-actions.spec.ts:191]: Yes. I think the menu change in `synth_r_2019q4_17` and this change _should_ get all tests passing again.

**Review by NehaBroad — APPROVED** [review]

**Pull request merged**